{"name": "st.family -- @1.1 transfer types", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not iterable\n", "trace": "  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\charity.py\", line 8, in step_impl\n    assert client.check_message_code(code)\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 110, in check_message_code\n    self.client.loop.run_until_complete(self.async_check_message_code(code))\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\asyncio\\base_events.py\", line 587, in run_until_complete\n    return future.result()\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 80, in async_check_message_code\n    for row in self.table:\n"}, "steps": [{"name": "Given client send message -/start-", "status": "passed", "start": 1605218646135, "stop": 1605218648760}, {"name": "And client click the button -Благодійність-", "status": "passed", "start": 1605218648762, "stop": 1605218651287}, {"name": "And client click the button -ST.FAMILY-", "status": "passed", "start": 1605218651288, "stop": 1605218653881}, {"name": "And client send message -запит-", "status": "passed", "start": 1605218653882, "stop": 1605218658982}, {"name": "And client click the button -Відмінити тікет-", "status": "passed", "start": 1605218658984, "stop": 1605218661719}, {"name": "When client click the button -Так-", "status": "passed", "start": 1605218661720, "stop": 1605218665132}, {"name": "Then client check message -Тікет скасований-1138-", "status": "broken", "statusDetails": {"message": "TypeError: 'NoneType' object is not iterable\n", "trace": "  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\charity.py\", line 8, in step_impl\n    assert client.check_message_code(code)\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 110, in check_message_code\n    self.client.loop.run_until_complete(self.async_check_message_code(code))\n  File \"c:\\users\\sasa\\anaconda3\\envs\\finbot\\lib\\asyncio\\base_events.py\", line 587, in run_until_complete\n    return future.result()\n  File \"C:\\Users\\sasa\\PycharmProjects\\FinBot\\base\\user_telegram.py\", line 80, in async_check_message_code\n    for row in self.table:\n"}, "start": 1605218665133, "stop": 1605218665663}], "parameters": [{"name": "button", "value": "ST.FAMILY"}], "start": 1605218646133, "stop": 1605218665671, "uuid": "21b2529c-640a-432f-a697-2a28cf06c2ca", "historyId": "59b4d465c53b2a663266ce74295d9132", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "sd_test12_bot"}, {"name": "feature", "value": "finbot_spd_charity"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}